container/split.go

```diff
@@ -60,6 +60,13 @@ func (s *Split) CreateRenderer() fyne.WidgetRenderer {
 	}
 }
 
+// ExtendBaseWidget is used by an extending widget to make use of BaseWidget functionality.
+//
+// Deprecated: Support for extending containers is being removed
+func (s *Split) ExtendBaseWidget(wid fyne.Widget) {
```

Comment:
I thought we said, on the call yesterday, that we wouldn't add "new" deprecated methods?